heron/common/src/java/com/twitter/heron/common/config/SystemConfig.java

```diff
@@ -131,6 +131,12 @@
       = "heron.instance.set.data.tuple.capacity";
 
   /**
+   * The maximum size in bytes of data tuple to batch in a HeronDataTupleSet protobuf
+   */
+  public static final String INSTANCE_SET_DATA_TUPLE_SIZE_BYTES
+      = "heron.instance.set.data.tuple.size.bytes";
```

Comment:
Would the word `batch` in here help clarify at all? `heron.instance.set.data.tuple.batch.size.bytes` for instance?